src/index.js

```diff
@@ -96,13 +96,38 @@ VERBS.concat('any').forEach(function(method) {
   };
 });
 
+function findInHandlers(method, handlers, handler) {
+  var index = -1;
+  for (var i = 0; i < handlers[method].length; i += 1) {
+    var item = handlers[method][i];
+    var isReplyOnce = item.length === 7;
+    var comparePaths = item[0] instanceof RegExp && handler[0] instanceof RegExp
+      ? deepEqual(String(item[0]), String(handler[0]))
+      : deepEqual(item[0], handler[0]);
```

Comment:
Since we're dealing with scalar values here, deepEqual doesn't need to be used.

Reply:
Addressed